### DISCUSSION TITLE
How to get github points for ibax

### DISCUSSION THREAD

**Discussion by mardi16** [IBAX-io/go-ibax#1582] [Q&A] [answered]
How to get github points for ibax

**Comment by Bulent3407** [comment]
İbax networke giriş yaptıktan sonra sağ üsteki pencereden github bağlantı seçiyorsun githuba bağlanıp yıldız ve çatal yapıyorsun

**Comment by yolandadadada** [comment, marked as answer]
You submit suggestions or bugs, accepted by IBAX, will be given points by IBAX.
